src/lib/lib.gni

```diff
@@ -29,4 +29,10 @@ declare_args() {
 
   # Set to true to enable device-specific attestation credentials
   chip_build_platform_attestation_credentials_provider = false
+
+  # Set to true to enable Trusty OS
```

Comment:
Please add actual description. If the option is named `chip_with_trusty_os` it obviously by reading it "enables chip to be built with Trusty OS".

So the comment should expand on this: what is trusty OS? Links to a documentation? What does it do why would anyone use it/enable it?

Something like "enables the use of Trusty: a NXP platform hardware encryption support library, allowing for faster/more secure/??? crypto operations. See http://.... for more information". Unsure if this is a library? Is it faster? Is it NXP specific? Is it android-only or available on freertos? Please add real documentation.

I made a suggestion below, but it still seems loose. At least it expands what Ele means based on what you wrote in the PR summary.